mmv1/templates/terraform/examples/integration_connectors_connection_advanced.tf.erb

```diff
@@ -0,0 +1,73 @@
+data "google_project" "test_project" {
+}
+
+resource "google_secret_manager_secret" "secret-basic" {
+  secret_id = "secret-version"
+  replication {
+    user_managed {
+      replicas {
+        location = "us-central1"
+      }
+    }
+  }
+}
+
+
+resource "google_secret_manager_secret_version" "secret-version-basic" {
+  secret = google_secret_manager_secret.secret-basic.id
+  secret_data = "Google@123@"
+}
+
+resource "google_secret_manager_secret_iam_member" "secret_iam" {
+  secret_id  = google_secret_manager_secret.secret-basic.id
+  role       = "roles/secretmanager.admin"
+  member     = "serviceAccount:${data.google_project.test_project.number}-compute@developer.gserviceaccount.com"
+  depends_on = [google_secret_manager_secret_version.secret-version-basic]
+}
+
+resource "google_integration_connectors_connection" "<%= ctx[:primary_resource_id] %>" {
+    name     = "<%= ctx[:vars]['connection_name'] %>"
+    description = "tf updated description"
+    service_account = "${data.google_project.test_project.number}-compute@developer.gserviceaccount.com"
+    connector_version = "projects/connectors-autopush-harsha/locations/global/providers/zendesk/connectors/zendesk/versions/1"
+    config_variable {
+        key = "proxy_enabled"
+        boolean_value = false
+    }
+    suspended = false
+    auth_config {
+      auth_type = "USER_PASSWORD"
+      user_password {
+        username = "balanaguharsha@google.com"
```

Comment:
I'd suggest that we change the values in the tests to not include any personal information since it's will be used in our web example and it's publicly visible.

Reply:
Changed